docs/CONTRIBUTING.md

```diff
@@ -109,19 +109,19 @@ sudo add-apt-repository \
 6. Update the apt package index again: `sudo apt-get update`
```

Comment:
Why are their changes to this file in your PR?  cc @manekenpix, @drwm-base, is this from your recent work?  Did it not get run through prettier (i.e., is this just a style fix that was missed in your merge)?

Reply:
After the first rebase, pr #398 was merged a few minutes after. It said there were conflicts with the md file in my branch and the one on master so I rebased.

Reply:
Must've been. Sorry about that, must have missed it. I had to rebase and it got a little messed up.